### DISCUSSION TITLE
Affine Error in Validation

### DISCUSSION THREAD

**Discussion by bryankhoo37** [MIC-DKFZ/nnUNet#1606] [Q&A]
Does anybody know why I am getting this error once nnunet begins validation on a fold? AssertionError: restored affine does not match original affine. There was no affine error popup when I preprocessed the data.

**Comment by simzhangbest** [comment]
me to ,mark !!!

AssertionError: restored affine does not match original affine

**Comment by wangnanv5** [comment]
you can use SimpleITKIO in dataset.json file to avoid this error
